remme/tp/atomic_swap.py

```diff
@@ -145,31 +150,15 @@ def _swap_init(self, context, signer_pubkey, swap_init_payload):
         swap_information.receiver_address = swap_init_payload.receiver_address
         swap_information.is_initiator = not swap_init_payload.secret_lock_by_solicitor
 
-        commission_amount = int(_get_setting_value(context, SETTINGS_SWAP_COMMISSION))
-        if commission_amount < 0:
-            raise InvalidTransaction('Wrong commission address.')
-
-        swap_total_amount = swap_information.amount + client_to_real_amount(commission_amount)
-
-        account = get_data(context, Account, swap_information.sender_address)
-
-        if account is None:
-            account = Account()
-
-        if account.balance < swap_total_amount:
-            raise InvalidTransaction(
-                f'Not enough balance to perform the transaction in the amount (with a commission) {swap_total_amount}.'
-            )
-
-        transfer_payload = AccountClient.get_transfer_payload(ConsensusAccountHandler.CONSENSUS_ADDRESS, commission_amount)
-
+        transfer_payload = AccountClient.get_transfer_payload(
```

Comment:
Why do so? We intentionally removed transferring to the zero address, this is an unnecessary overhead.

Reply:
Here is required a transfer from one address to another, because it is not a fee and we do not need it on block cost.